UPDATING.md

```diff
@@ -5,6 +5,35 @@ assists people when migrating to a new version.
 
 ## Airflow Master
 
+### New Webserver UI with Role-Based Access Control (alpha version)
+
+Our current webserver UI uses the Flask-Admin extension. The new webserver UI uses the [Flask-AppBuilder (FAB)](https://github.com/dpgaspar/Flask-AppBuilder) extension. It has built-in authentication support and Role-Based Access Control (RBAC), which provides configurable roles and permissions for individual users.
+
+To turn on this feature, in your airflow.cfg file, under [webserver], set configuration variable `rbac = True`, and then run `airflow` command, which will generate the `webserver_config.py` file in your $AIRFLOW_HOME.
+
+#### Setting up Authentication
+
+FAB has built-in authentication support for DB, OAuth, OpenID, LDAP, and REMOTE_USER. The default auth type is `AUTH_DB`.
+
+For any other authentication type (OAuth, OpenID, LDAP, REMOTE_USER), see [this link](http://flask-appbuilder.readthedocs.io/en/latest/security.html#authentication-methods) for how to configure variables in webserver_config.py file.
```

Comment:
Try to avoid having link targets saying "this link" or "click here", as it doesn't provide clues for screen readers and other assistive technologies. (I guess it doesn't matter much in this specific case, but it is a good habit to get in to.)

Something like:

```see the [Authentication section of FAB docs](http://flask-appbuilder.readthedocs.io/en/latest/security.html#authentication-methods)```